code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -147,6 +147,27 @@
 	gas_type = GAS_FREON
 	filled = 1
 
+/obj/machinery/portable_atmospherics/canister/hydrogen
+	name = "hydrogen canister"
+	desc = "Hydrogen. Used in thruster fuel"
+	icon_state = "yellow"
+	gas_type = GAS_HYDROGEN
+
+/obj/machinery/portable_atmospherics/canister/fuel
+	name = "fuel canister"
+	desc = "A highly volatile mix of hydrogen and oxygen."
+	icon_state = "red"
```

Comment:
might be nice to ask a spriter for unique sprites for these two? fine if not

Reply:
made some myself!